src/GafferArnold/ArnoldDisplacement.cpp

```diff
@@ -181,8 +181,10 @@ IECore::ConstCompoundObjectPtr ArnoldDisplacement::attributes( const Gaffer::Plu
 	m[g_heightAttributeName] = new FloatData( heightPlug()->getValue() );
 	m[g_paddingAttributeName] = new FloatData( paddingPlug()->getValue() );
 	m[g_zeroValueAttributeName] = new FloatData( zeroValuePlug()->getValue() );
-	m[g_autoBumpAttributeName] = new BoolData( autoBumpPlug()->getValue() );
-
+	if ( autoBumpPlug()->getValue() )
```

Comment:
Testing `!autoBumpPlug()->isSetToDefault()` would be better here too I think, so that if someone was using the plug explicitly (via expression or whatever) we still output the attribute, even if the value is false.

Technically anything we do here will be a breaking change because scene output will change. If this needs to get into `0.60` I think I'm OK with making an exception because I'd presume the likelihood of it mattering is infinitessimal, but do you have any concerns @andrewkaufman, @danieldresser-ie?